app/components/graph-editor/canvas.ts

```diff
@@ -269,14 +295,19 @@ export function setLineStyle(
     value: string,
     dotSize?: number
 ) {
-    if (!dotSize)
-        dotSize = g.lineWidth;
-    if (value == "dashed")
-        g.setLineDash([3 * dotSize, 6 * dotSize]);
-    else if (value == "dotted")
-        g.setLineDash([dotSize, 2 * dotSize]);
-    else
-        g.setLineDash([1, 0]);
+    dotSize = (dotSize ? dotSize : g.lineWidth);
+    switch (value) {
```

Comment:
`value` is supposed to come straight off of a graph element. I'm not exactly sure what the advantage would be of making value a function that returns the appropriate array for `g.setLineDash`. The function would still have a switch or if/else statement in it.